src/containers/shared/components/Transaction/XChainAddAttestation/ClaimAttestation.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import Account from '../../Account'
+import { Amount } from '../../Amount'
+import { SimpleRow } from '../SimpleRow'
+
+export interface ClaimAttestationProps {
+  send: string
+  account: string
+  destination: string
+  claimId: string
+  signature: string
+}
+
+export const ClaimAttestation = (props: ClaimAttestationProps) => {
+  const { t } = useTranslation()
+  const { send, account, destination, claimId, signature } = props
+
+  return (
+    <div className="attestation" key={signature}>
+      <div className="attestation-title">
+        XChain Claim Attestation Batch Element
```

Comment:
i18n is needed here.

Reply:
Is that needed for all text on the website? Or is there a specific reason this one needs it? (Just trying to understand so I can apply the same logic to other PRs when I'm reviewing :)

Reply:
I think every place should use it.  Doesn't really make sense to have some of it capable of being translated.  Makes it difficult to track down later when we support more languages.